charts/gateway/templates/deployment.yaml

```diff
@@ -413,11 +413,6 @@ spec:
           - name: OTEL_RESOURCE_ATTRIBUTES
             value: {{ template "gateway.otel.resource.attributes" . }}
           {{- end }}
-          env:
```

Comment:
this part is probably from original gemfire work. 
it is not found anywhere in stable or develop/gateway branch